mvc/controllers/patchVote.js

```diff
@@ -0,0 +1,20 @@
+const putVote = require("../models/putVote.js");
+
+const patchVote = async (req, res, next) => {
+  if (req.body.inc_votes) {
+    const patch = await putVote(req.body, req.params.review_id).catch((err) => {
+      next(err);
```

Comment:
you need to use try catch here

Reply:
wont the .catch on the end do? I read somewhere its the ideal format for async ??

Reply:
done